src/rdkafka_admin.c

```diff
@@ -6217,6 +6285,23 @@ const char *rd_kafka_ConsumerGroupDescription_partition_assignor(
         return grpdesc->partition_assignor;
 }
 
+size_t rd_kafka_ConsumerGroupDescription_authorized_operation_count(
+    const rd_kafka_ConsumerGroupDescription_t *grpdesc) {
+        if (grpdesc->authorized_operations)
+                return rd_list_cnt(grpdesc->authorized_operations);
+        return 0;
+}
+
+rd_kafka_AclOperation_t rd_kafka_ConsumerGroupDescription_authorized_operation(
+    const rd_kafka_ConsumerGroupDescription_t *grpdesc,
+    size_t idx) {
+        if (grpdesc->authorized_operations) {
+                rd_kafka_AclOperation_t *entry =
+                    rd_list_elem(grpdesc->authorized_operations, idx);
+                return *entry;
```

Comment:
We should not do this for any case, since the user knows already what the count is (it's similar to the idea of 'Don’t check for NULLs, if parameters are required it’s caller responsibility to pass them'). 
We should even remove the `if (grpdesc->authorized_operations)` in my opinion, what do you think?
See for example, rd_kafka_ConsumerGroupDescription_member